CMakeLists.txt

```diff
@@ -51,13 +51,15 @@ option(BUILD_TORCHTEXT_PYTHON_EXTENSION "Build Python extension" OFF)
 
 set(CMAKE_MODULE_PATH "${CMAKE_MODULE_PATH};${CMAKE_CURRENT_SOURCE_DIR}/cmake")
 
-find_package(Torch REQUIRED)
+set(TORCH_INSTALL_PREFIX $ENV{TORCH_INSTALL_PREFIX})
```

Comment:
I think this should be an CMake option and being passed from [`setup.py`](https://github.com/pytorch/text/blob/main/tools/setup_helpers/extension.py#L62). So that the logic is hidden from user.

Reply:
Done. Lmk if the changes look okay